README.md

```diff
@@ -24,7 +24,7 @@
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
@HAMMAS-SALEEM  Thanks for your feedback I always appreciate your review. I will work on the logo soon.